app/backend/src/dummy_data.py

```diff
@@ -348,3 +353,8 @@ def add_dummy_communities():
 def add_dummy_data():
     add_dummy_users()
     add_dummy_communities()
+
+
+def test_add_dummy_data(db, caplog):
+    add_dummy_data()
+    assert len(caplog.records) == 0
```

Comment:
Why not put this in `tests/`?

Reply:
Good question! I somehow thought that `tests/` should only test the `couchers/` code.

(But one could equally well say that 'dummy_data.py` should run without the `tests/` code.)

What do you think?

Reply:
I was just thinking of `tests/` as where the tests go :)

Or maybe just move it into `test_dummy_data.py` next to `dummy_data.py`? I just feel mixing testing code with the main code is a bit icky?

Reply:
Ok, I both made an own file and moved it to tests :) Let's see if CI picks it up.